.env

```diff
@@ -0,0 +1,2 @@
+EARTHLY_ALLOW_PRIVILEGED=true
+EARTHLY_INSTALLATION_NAME=translate
```

Comment:
I suggest inserting empty line at the EOF, it can be done automatically in vscode:
![image](https://user-images.githubusercontent.com/69954124/216293828-d584f56f-885a-4e65-82a1-db5d88910500.png)
I am sure that jetbrains products have something similar
It is not mandatory but is a good practice, [explanation](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)